Rails/rails_tutorial.md

```diff
@@ -77,24 +95,24 @@ Ví dụ văn phòng Đà Nẵng là box: "[edu][DN] Pull Request's Status Check
 ## 7. Thiết lập Sublime Text
 Các bạn sử dụng setting này dành cho sublime để xử lý indent, trailing space và end of file, bằng cách:
 Preference -> Setting User -> xóa hết cái cũ và paste đoạn code sau vào:
-```
+```yaml
 {
-	"ensure_newline_at_eof_on_save": true,
-	"font_size": 11,
-	"highlight_line": true,
-	"ignored_packages":
-	[
-		"Vintage"
-	],
-	"rulers":
-	[
-		80
-	],
-	"show_encoding": true,
-	"show_line_endings": true,
-	"tab_size": 2,
-	"translate_tabs_to_spaces": true,
-	"trim_trailing_white_space_on_save": true
+  "ensure_newline_at_eof_on_save": true,
+  "font_size": 11,
```

Comment:
```suggestion
  "font_size": 14,
```
anh đề xuất nên để mặc định là `14`, sau đó ai muốn to nhỏ gì thì tùy chỉnh.
để 11 bé tí, sẽ hơi hại mắt các bạn trainee.
mà hầu hết các bạn ấy lại còn đeo kính nữa chứ :-ss :v